src/components/posts/PostHeader.js

```diff
@@ -12,8 +12,8 @@ function PostHeader(props) {
       />
       <div className="post-header text-bold ">{props.header}</div>
 
-      <div className="post-subheader body-text text-bold ">
-        {props.subheader}
+      <div className="post-subheader body-text">
+        <i>{props.subheader}</i>
```

Comment:
This seems pretty nitpicky from me, but I would avoid using HTML tags for pure styling (in this case, you just want `props.subheader` to be italicized). HTML is usually for semantics and meaning, while CSS is for styling. I'd plop italics into `.post-subheader` or something similar!